src/styles/toggle-switch.scss

```diff
@@ -80,8 +84,8 @@ $toggle-height: 25px;
     box-shadow: 2px 2px 5px 0 rgba($med-gray, 0.4);
     display: inline-block;
     height: $toggle-height;
-    margin-left: calc(#{$toggle-height} * -2.5);
-    margin-top: 3px;
+    margin-left: calc(#{$toggle-height} * -2.4);
+    margin-top: 2px;
```

Comment:
Are style changes necessary? We should try to avoid them if possible.

Reply:
This came from a bug where the parent elements were smaller than the child -- it looks the same, but the parents' sizes are accurate now.